WispRoom.rb

```diff
@@ -108,7 +89,7 @@ def dropWisps
   end
 
   def get_action
-    input = Game.get_input
+    input = get_input
```

Comment:
Since this file is included by the file that defines `Game` (namely `leeroy.rb`), you can leave this line as
```ruby
input = Game.get_input
```